src/compiler/diagnosticMessages.json

```diff
@@ -1859,10 +1859,22 @@
         "category": "Error",
         "code": 2553
     },
-    "Expected {0} type arguments, but got {1}.": {
+    "Expected at least {0} arguments, but got {1}.": {

```

Comment:
I think the diff is wonky here. All these messages are new, so there aren't any codes to reuse.